app/cdap/components/WranglerV2/SearchResultsListItem/SearchResultListItem.stories.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { action } from '@storybook/addon-actions';
+import SearchResultListItem from 'components/WranglerV2/SearchResultsListItem';
+
+export default {
+  title: 'SearchResultListItem',
+  component: SearchResultListItem,
+};
+
+const handleClick = () => action('clicked')('Search item clicked');
+
+export function Default(args) {
```

Comment:
can u rename the components to with icon and without icon specifics to make them more meaningful

Reply:
Done